### PR TITLE
Fix kubectl config file delimiter on windows

### PR DESCRIPTION
**Pull request by Tryneus**
`pachctl deploy` commands fail on Windows because `:` is not a valid path delimiter.

### CODE REVIEW

File: src/server/pkg/deploy/cmds/cmds.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func kubectl(stdin io.Reader, context *config.Context, args ...string) error {
 				return errors.Wrapf(err, "failed to discover default kube config: %q does not exist", kubeconfig)
 			}
 		}
-		kubeconfig = fmt.Sprintf("%s:%s", kubeconfig, tmpfile.Name())
+		kubeconfig = fmt.Sprintf("%s;%s", kubeconfig, tmpfile.Name())
```
Comment (ysimonson) [inline src/server/pkg/deploy/cmds/cmds.go:101]: This'll break on unices. We need conditional logic to use `;` on windows, and `:` everywhere else

Comment (jdoliner) [reply src/server/pkg/deploy/cmds/cmds.go:101]: `os.PathListSeparator` will contain the correct separator for the operating system you're compiling for.

Comment (Tryneus) [reply src/server/pkg/deploy/cmds/cmds.go]: Riiight - fixed

File: src/server/pkg/deploy/cmds/cmds.go [hunk 2]
```diff
@@ -98,7 +98,7 @@ func kubectl(stdin io.Reader, context *config.Context, args ...string) error {
 				return errors.Wrapf(err, "failed to discover default kube config: %q does not exist", kubeconfig)
 			}
 		}
-		kubeconfig = fmt.Sprintf("%s:%s", kubeconfig, tmpfile.Name())
+		kubeconfig = kubeconfig + string(os.PathListSeparator) + tmpfile.Name()
```
Comment (ysimonson) [inline src/server/pkg/deploy/cmds/cmds.go:101]: Nitpick, but would you mind using `fmt.Sprintf`? It should be a bit faster.

**Pull request merged**